Google.PowerShell/Common/GcloudCmdlet.cs

```diff
@@ -43,7 +43,7 @@ public GCloudCmdlet()
         /// <summary>
         /// Returns an instance of the Google Client API initializer, using the machine's default credentials.
         /// </summary>
-        protected BaseClientService.Initializer GetBaseClientServiceInitializer()
+        protected static BaseClientService.Initializer GetBaseClientServiceInitializer()
```

Comment:
Can you back this change out? It seems unrelated to creating a new `Copy-GcsObject` cmdlet, and we should talk about the design implications here.